bigtable/tests/system.py

```diff
@@ -210,33 +209,118 @@ def test_update(self):
         Config.INSTANCE.update()
 
     def test_create_app_profile_with_multi_routing_policy(self):
-        # Create a new instance instance and reload it.
+        from google.cloud.bigtable_admin_v2.types import instance_pb2
+
         description = 'Foo App Profile'
         instance = Config.INSTANCE
+        ignore_warnings = True
+        app_profile_id = 'app_profile_id_1'
 
         app_profile = instance.create_app_profile(
-            app_profile_id=APP_PROFILE_ID+'-multi',
+            app_profile_id=app_profile_id,
             routing_policy_type=ROUTING_POLICY_TYPE_ANY,
             description=description,
-            ignore_warnings=True
+            ignore_warnings=ignore_warnings
         )
 
-        self.assertEqual(app_profile.description, description)
+        # Load a different app_profile objec form the server and
+        # verrify that it is the same
+        alt_app_profile = instance.get_app_profile(app_profile_id)
+        self.assertEqual(app_profile, alt_app_profile)
+
+        # Modify existing app_profile to singly routing policy and confirm
+        new_description = 'To single routing policy'
+        allow_transactional_writes = False
+        operation = instance.update_app_profile(
+            app_profile_id=app_profile_id,
+            routing_policy_type=ROUTING_POLICY_TYPE_SINGLE,
+            description=new_description,
+            ignore_warnings=ignore_warnings,
```

Comment:
For completeness, should we be updating `ignore_warnings` to False here?

Reply:
In general, it's a good idea to `ignore_warnings=True` in code.  `ignore_warnings=False` only seems to make sense for UI purposes.

Reply:
OK.